### PR TITLE
Avoid unnecessary updates in `F.batch_renormalization`, and related fixes

### PR DESCRIPTION
**Pull request by toslunar**
Each commit fixes a minor error.  The biggest change is to exclude `eps` from `running_var`.

### CODE REVIEW

**Comment by niboshi** [comment]
How about separating a PR for the non-trivial fix?

**Comment by toslunar** [comment]
Removed the changes on `eps` from the PR.  See the diff of the force-push.

File: chainer/functions/normalization/batch_renormalization.py [hunk 1]
```diff
@@ -173,21 +171,38 @@ def batch_renormalization(x, gamma, beta, rmax, dmax, eps=2e-5,
...
-        with `update_statistics=True` option.
+        with ``update_statistics=True`` option.
+
+    .. note::
+
+        For the consistency with Batch Normalization, this function
+        intentionally ignores some of the theoretical flaws in Algorithm 1 of
+        the Batch Renormalization paper:
+
+        - The function maintains the moving average of variances
```
Comment (niboshi) [inline chainer/functions/normalization/batch_renormalization.py]: A nitpicking suggestion that makes the sentence more clear while reading.

```suggestion
        - ``F.batch_renormalization`` maintains the moving average of variances
```

File: chainer/functions/normalization/batch_renormalization.py [hunk 2]
```diff
@@ -173,21 +171,38 @@ def batch_renormalization(x, gamma, beta, rmax, dmax, eps=2e-5,
...
+          :math:`\\sigma^2`, while the original paper maintain the moving
```
Comment (niboshi) [inline chainer/functions/normalization/batch_renormalization.py]:
```suggestion
          :math:`\\sigma^2`, while the original paper maintains the moving
```

File: chainer/functions/normalization/batch_renormalization.py [hunk 3]
```diff
@@ -173,21 +171,38 @@ def batch_renormalization(x, gamma, beta, rmax, dmax, eps=2e-5,
...
+          average of standard deviations :math:`\\sigma`.
+        - The function applies Bessel's correction to update the moving average
```
Comment (niboshi) [inline chainer/functions/normalization/batch_renormalization.py]: Ditto.
```suggestion
        - ``F.batch_renormalization`` applies Bessel's correction to update the moving average
```

**Comment by niboshi** [comment]
Please fix test error.

**Comment by toslunar** [comment]
Fixed.

**Comment by niboshi** [comment]
LGTM.
Jenkins, test this please.

**Pull request merged**